stroyprombeton/management/commands/test_db.py

```diff
@@ -39,6 +39,11 @@
 SERIES_PATTERN = 'Series #{}'
 EMPTY_SERIES_PATTERN = 'Series empty #{}'
 
+REAL_SECTIONS_COUNT = 2
```

Comment:
```suggestion
FILLED_SECTIONS_COUNT = 2
```
`REAL_` prefix is too general, so it doesn't bring any info about sections